bob2.py

```diff
@@ -0,0 +1,71 @@
+import struct
+import socket
+
+class Bob2Protocol:
+    def __init__(self, version_major=0, version_minor=0):
+        self.version_major = version_major
+        self.version_minor = version_minor
+
+    def build_message(self, message_type, dest_ipv6, dest_port, message_content):
```

Comment:
Is there a reason we use ipv6? Our raspberry pi's don't have ipv6 addresses, right?

Reply:
These addresses aren't going to involve the pis - they are the "earth" addresses, so i just used ipv6 in a trying to use modern standards way :)